examples/src/test/resources/examples/datasource_multiple_maps/expected_output/image-similarity.txt

```diff
@@ -1 +1 @@
-175
+190
```

Comment:
I'm afraid so. Maybe the fonts, maybe the java version, maybe the OS. We used to have different expected images for windows. Now we don't care about windows builds...